framework/source/class/qx/theme/manager/Decoration.js

```diff
@@ -122,8 +122,11 @@ qx.Class.define("qx.theme.manager.Decoration",
       // create and add a CSS rule
       var css = "";
       var styles = instance.getStyles(true);
-      for (var key in styles) {
-
+      
+      // Sort the styles so that more specific styles come after the group styles, 
+      //  eg background-color comes after background
+      var t = this;
+      Object.keys(styles).sort().forEach(function(key) {
```

Comment:
By the way, I would leave it with the original for loop and just reorder the specific broken ones for now, to fix this bug. We should then change it to the sorted list at the next major version.